tests/model/test_ocrd_mets.py

```diff
@@ -1,265 +1,364 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is that necessary/does it help?

Reply:
Well, I've found this in many places with Beginner Tutorials etc. It's because to get an alert on other OS where UTF-8 is not set as default charset (windows, for example).

Reply:
I always thought this was an emacs convention, like vim's modelines but was confused since I had you down as a VisualCode user :)